cms/djangoapps/contentstore/tests/test_import.py

```diff
@@ -283,3 +283,49 @@ def _verify_split_test_import(self, target_course_name, source_course_name, spli
         }
 
         self.assertEqual(remapped_verticals, split_test_module.group_id_to_child)
+
+    @ddt.data(ModuleStoreEnum.Type.mongo, ModuleStoreEnum.Type.split)
+    def test_ora_rubric_settings_import_draft_correctly(self, store):
+        """
+        Test that questions/rubrics/settings are correctly import for ora draft problems
+        """
+        with modulestore().default_store(store):
+            module_store = modulestore()
+            course_id = module_store.make_course_key('edX', 'test_draft_settings_ora', '2015_AN')
+            import_course_from_xml(
+                module_store,
+                self.user.id,
+                TEST_DATA_DIR,
+                ['2015_A20'],
```

Comment:
I agree (non-nit). The test courses should all have a name that suggests what it is they test.